### PR TITLE
[Bug 1285306] Increase admin node storage

### PR DESCRIPTION
**Pull request by flamingspaz**
- Launch node from AMI with smaller storage
- Increase root EBS volume to 20GB

### CODE REVIEW

File: admin.tf [hunk 1]
```diff
@@ -49,14 +49,20 @@ resource "aws_security_group_rule" "admin-ec2-sg-allowallegress" {
 }
 
 resource "aws_instance" "admin-ec2" {
-    ami                     = "${lookup(var.aws_amis, var.aws_region)}"
+#   ami                     = "${lookup(var.aws_amis, var.aws_region)}"
+    ami                     = "ami-68d8557f"  # For bug 1285306
```
Comment (johngian) [inline admin.tf]: Let's add a variable for that instead of hardcoded value.

Comment (johngian) [inline admin.tf]: eg. ``admin-node-ami``

**Comment by johngian** [comment]
r+wc

**Comment by flamingspaz** [comment]
I used the same name as the AMI in AWS to make it consistent.

File: admin.tf [hunk 2]
```diff
@@ -49,14 +49,20 @@ resource "aws_security_group_rule" "admin-ec2-sg-allowallegress" {
 }
 
 resource "aws_instance" "admin-ec2" {
-    ami                     = "${lookup(var.aws_amis, var.aws_region)}"
+#   ami                     = "${lookup(var.aws_amis, var.aws_region)}"
```
Comment (johngian) [inline admin.tf]: Please delete this comment.

File: admin.tf [hunk 3]
```diff
@@ -49,14 +49,20 @@ resource "aws_security_group_rule" "admin-ec2-sg-allowallegress" {
...
+    ami                     = "${lookup(var.aws_amis, "admin-node-2016-07-15")}"  # For bug 1285306
```
Comment (johngian) [inline admin.tf]: I am not sure if the nested ``"`` is going to work here.

**Pull request merged**